src/components/Callout/Callout.md

```diff
@@ -3,7 +3,6 @@
 Default hover trigger:
 
 ```js { "props": { "data-example": "default hover trigger" } }
-const { DirectionalHint } = require('.');
```

Comment:
is this not working anymore?

Reply:
@chrismohr points it out in the description of the PR. They've gone from normal `enum`s to `const enum`s, which apparently don't generate JavaScript. They're citing reduced bundle size as the reason. So now we cannot use their constants from JavaScript!

Reply:
As a solution, I was thinking of using the `updateExample` property, extracting all examples to their own separate files, and compiling them with `tsc`. That'd also solve the issue you cited of not being able to use your IDE to write examples. Whatchuthink?

Reply:
@unindented are you suggesting that as part of this PR or as a follow-up item?

Reply:
Let's say followup. I don't want this to be blocked.

Reply:
https://github.com/Microsoft/YamUI/issues/131